README.md

```diff
@@ -23,3 +24,5 @@ inefficient.
 
 What is the big $\Theta$ complexity of your implementation? Add your
 answer, including your reasoning, to this markdown file.
+
+- The time complexity of my implementation is $\Theta(e∗log(v))$, where e is the number of edges in a graph and v is the number of vertices. First we iterate through all nodes in the unvisited set which takes $O(v)$ time. The main loop then iterates at most v times. Finding the closest node within unvisited using a priority queue means a worst case of $O\log(v)$, and then updating the neighbors contributes $O(e)$ in the worst case. That makes a complexity of  $v+v(log(v))+e$ which can be bounded by $\Theta(e∗log(v)).$
```

Comment:
Not following you here -- you've got a loop that iterates over all nodes, so it couldn't be less than that.

Reply:
Sorry, about that! I have made the necessary changes; in my updated analysis, I said that for any given node it can at most v-1 edges (where v is the number of vertices) assuming we are given a graph with no redundant edges. Please let me know if this is not something that I can assume.